validator-api/Cargo.toml

```diff
@@ -22,24 +23,34 @@ log = "0.4"
 pin-project = "1.0"
 pretty_env_logger = "0.4"
 rand = "0.7"
-reqwest = { version="0.11", features=["json"] }
-rocket = { version="0.5.0-rc.1", features=["json"] }
+reqwest = { version = "0.11", features = ["json"] }
+rocket = { version = "0.5.0-rc.1", features = ["json"] }
 serde = "1.0"
 serde_json = "1.0"
-tokio = { version="1.4", features=["rt-multi-thread", "macros", "signal", "time"] }
-rocket_cors = { git="https://github.com/lawliet89/rocket_cors", branch="master" }
+tokio = { version = "1.4", features = [
+    "rt-multi-thread",
+    "macros",
+    "signal",
+    "time",
+] }
+rocket_cors = { git = "https://github.com/lawliet89/rocket_cors", branch = "master" }
```

Comment:
I thought I already merged a PR that changed cors dependency to point to specific commit instead?